pymathics/natlang/linguistic_data.py

```diff
@@ -0,0 +1,351 @@
+# -*- coding: utf-8 -*-
+"""
+Linguistic Data
+
+See <url>:WMA:https://reference.wolfram.com/language/guide/LinguisticData.html</url> guide.
+
+"""
+# This module uses both nltk and spacy. Maybe it makes sense to split this further.
+
+
+# TODO: Complete me
+
+# WordFrequencyData — data on typical current and historical word frequencies
+# Synonyms — synonyms for a word
+# Antonyms — antonyms for a word
+# PartOfSpeech — possible parts of speech for a word
+
+
+from typing import Optional
+
+from mathics.builtin.base import Builtin, MessageException
+
+# from mathics.builtin.codetables import iso639_3
+from mathics.builtin.numbers.randomnumbers import RandomEnv
+from mathics.core.atoms import String
+from mathics.core.convert.expression import Expression, to_expression
+from mathics.core.evaluation import Evaluation
+from mathics.core.list import ListExpression
+from mathics.core.symbols import Symbol, SymbolList
+from mathics.core.systemsymbols import SymbolMissing, SymbolRule, SymbolStringExpression
+from pattern.en import pluralize
+
+from pymathics.natlang.textual_analysis import WordStem
+from pymathics.natlang.util import (
+    WordProperty,
+    _WordListBuiltin,
+    _wordnet_pos_to_type,
+    _WordNetBuiltin,
+    merge_dictionaries,
+)
+
+SymbolDictionaryLookup = Symbol("Pymathics`Natlang`DictionaryLookup")
+StringNotAvailable = String("NotAvailable")
+
+
+class Pluralize(Builtin):
+    """
+    <url>:WMA:
+    https://reference.wolfram.com/language/ref/Pluralize.html</url>
+
+    <dl>
+      <dt>'Pluralize[$word$]'
+      <dd>returns the plural form of $word$.
+    </dl>
+
+    >> Pluralize["potato"]
+     = potatoes
+    """
+
+    requires = ("pattern",)
+    summary_text = "Retrieve the pluralized form of a word"
```

Comment:
Retrieve -> retrieve and the other places in summary text below. (A couple are already lowercase).